crate/kmip/src/kmip/kmip_objects.rs

```diff
@@ -150,6 +150,7 @@ impl Object {
     }
 
     /// Return the `KeyWrappingData` of that object if any
+    #[must_use]
```

Comment:
This function is not expensive, is it really necessary?

Reply:
I must say I strictly apply clippy pedantic warnings:
```
error: this method could have a `#[must_use]` attribute
   --> crate/kmip/src/kmip/kmip_objects.rs:153:5
    |
153 |     pub fn key_wrapping_data(&self) -> Option<&KeyWrappingData> {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: add the attribute: `#[must_use] pub fn key_wrapping_data(&self) -> Option<&KeyWrappingData>`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#must_use_candidate
    = note: `-D clippy::must-use-candidate` implied by `-D warnings`

```